components/dashboard/src/org-admin/AdminPage.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2025 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import React, { useEffect } from "react";
+import { useHistory } from "react-router-dom";
+import { useUserLoader } from "../hooks/use-user-loader";
+import { useCurrentOrg } from "../data/organizations/orgs-query";
+import { useIsOwner } from "../data/organizations/members-query";
+import Header from "../components/Header";
+import { SpinnerLoader } from "../components/Loader";
+import { RunningWorkspacesCard } from "./RunningWorkspacesCard";
+import { MaintenanceModeCard } from "./MaintenanceModeCard";
+import { MaintenanceNotificationCard } from "./MaintenanceNotificationCard";
+
+const AdminPage: React.FC = () => {
+    const history = useHistory();
+    const { loading: userLoading } = useUserLoader();
+    const { data: currentOrg, isLoading: orgLoading } = useCurrentOrg();
+    const isOwner = useIsOwner();
+
+    useEffect(() => {
+        if (userLoading || orgLoading) {
+            return;
+        }
+        if (!isOwner) {
+            history.replace("/workspaces");
+        }
+    }, [isOwner, userLoading, orgLoading, history, currentOrg?.id]);
+
+    return (
+        <div className="flex flex-col w-full">
+            <Header title="Organization Administration" subtitle="Manage Infrastructure Rollouts." />
+            <div className="app-container py-6">
+                <h2 className="text-2xl font-semibold text-pk-content-primary mb-4">Infrastructure Rollout</h2>
```

Comment:
This section feels a bit cluttered. Let's add a bit more spacing.

```suggestion
            <div className="app-container py-6 flex flex-col gap-4">
                <h2 className="text-2xl font-semibold text-pk-content-primary">Infrastructure Rollout</h2>
```